src/base/api.lua

```diff
@@ -76,12 +76,15 @@
 ---
 -- Register a general-purpose includeExternal() call which works just like
 -- include(), but marks any containers created while evaluating the included
--- scripts as external.
+-- scripts as external. It also, loads the file regardless of how many times
+-- it has been loaded already.
 ---
 
 	function includeexternal(fname)
+		local fullPath = os.locate(fname, fname .. ".lua", path.join(fname, "premake5.lua"), path.join(fname, "premake4.lua"))
```

Comment:
Tests are always appreciated! Maybe put them in `tests/base/test_include.lua`? If I were trying to find them, that is probably the first place I'd look.